openedx/core/djangoapps/discussions/models.py

```diff
@@ -90,18 +96,18 @@ def pii_sharing_required_message(provider_name):
 AVAILABLE_PROVIDER_MAP = {
     'legacy': {
         'features': [
-            Features.ANONYMOUS_POSTING.value,
-            Features.BLACKOUT_DISCUSSION_DATES.value,
-            Features.COMMUNITY_TA_SUPPORT.value,
-            Features.COURSE_COHORT_SUPPORT.value,
-            Features.DISCUSSION_PAGE.value,
-            Features.INTERNATIONALIZATION_SUPPORT.value,
-            Features.PRIMARY_DISCUSSION_APP_EXPERIENCE.value,
-            Features.QUESTION_DISCUSSION_SUPPORT.value,
-            Features.RESEARCH_DATA_EVENTS.value,
-            Features.REPORT_FLAG_CONTENT_TO_MODERATORS.value,
-            Features.WCAG_2_0_SUPPORT.value,
-            Features.WCAG_2_1.value,
+            Features.ANONYMOUS_POSTING.name,
```

Comment:
I think the first argument is more like an id than a name, what are your thoughts on this?